partials/content.hbs

```diff
@@ -98,18 +81,25 @@
                 {{#if @custom.show_share_links}}
                     <div class="single-footer-bottom">
                         <div class="share">
-                            <a class="share-link share-link-facebook" href="https://www.facebook.com/sharer.php?u={{url absolute="true"}}" target="_blank" rel="noopener noreferrer">
+                            <a class="share-link share-link-facebook" id="facebookShareButton" href="https://www.facebook.com/sharer.php?u={{url absolute="true"}}" target="_blank" rel="noopener noreferrer">
                                 <svg class="icon icon-facebook">
                                     <use xlink:href="#facebook"></use>
                                 </svg>
                                 Share
                             </a>
-                            <a class="share-link share-link-twitter" href="https://twitter.com/intent/tweet?url={{url absolute="true"}}&text={{encode title}}" target="_blank" rel="noopener noreferrer">
+                            <a class="share-link share-link-twitter" id="twitterShareButton" href="https://twitter.com/intent/tweet?url={{url absolute="true"}}&text={{encode title}}" target="_blank" rel="noopener noreferrer">
                                 <svg class="icon icon-twitter">
-                                    <use xlink:href="#twitter"></use>
-                                </svg>
+                                        <use xlink:href="#twitter"></use>
```

Comment:
Seems like extra indentation was added here, please revert back to the indentation of the other icons!